.github/instructions/contributing.instructions.md

```diff
@@ -116,55 +120,17 @@ canonical: /path/to/canonical/doc/
 # OR
 
 canonical: /{{< latest "influxdb" "v2" >}}/path/to/canonical/doc/
-```
-
-### v2 equivalent documentation
-
-To display a notice on a 1.x page that links to an equivalent 2.0 page,
-add the following frontmatter to the 1.x page:
-
-```yaml
-v2: /influxdb/v2.0/get-started/
-```
-
-### Alternative links for cross-product navigation
-
-Use the `alt_links` frontmatter to specify equivalent pages in other InfluxDB products,
-for example, when a page exists at a different path in a different version or if
-the feature doesn't exist in that product.
-This enables the product switcher to navigate users to the corresponding page when they
-switch between products. If a page doesn't exist in another product (for example, an
-Enterprise-only feature), point to the nearest parent page if relevant.
-
-```yaml
+yaml
 alt_links:
   cloud-dedicated: /influxdb3/cloud-dedicated/admin/tokens/create-token/
   cloud-serverless: /influxdb3/cloud-serverless/admin/tokens/create-token/
   core: /influxdb3/core/reference/cli/influxdb3/update/  # Points to parent if exact page doesn't exist
-```
-
-Supported product keys for InfluxDB 3:
-- `core`
-- `enterprise`
-- `cloud-serverless`
-- `cloud-dedicated`
-- `clustered`
-
-### Prepend and append content to a page
-
-Use the `prepend` and `append` frontmatter to add content to the top or bottom of a page.
-Each has the following fields:
-
-```yaml
-append: |
-  > [!Note]
-  > #### This is example markdown content
-  > This is just an example note block that gets appended to the article.
-```
-
-Use this frontmatter with [cascade](#cascade) to add the same content to
-all children pages as well.
-
+yaml
+cascade:
+  append: |
+    > [!Note]
+    > #### This is example markdown content
+    > This is just an example note block that gets appended to the article.
 md
```

Comment:
Not sure what happened here, but the codeblock formatting seems to have broken things.